libs/datamodel/core/tests/types/mod.rs

```diff
@@ -1,2 +1,3 @@
+mod mysql_native_types;
```

Comment:
Do the tests actually run when this is not `pub`? 🤔 
I really don't know. I simply followed the existing patterns :-D

Reply:
they do, yes 😄

Reply:
but for consistency I can add the `pub` here